frontend/cssMock.ts

```diff
@@ -0,0 +1 @@
+export default '';
```

Comment:
Hi @Bhavin789 
Can you please help me here why we are exporting an empty string